packages/loader/src/gltf/parser/GLTFParserContext.ts

```diff
@@ -107,9 +115,31 @@ export class GLTFParserContext {
           this._createAnimator(this, glTFResource.animations);
         }
         this.resourceManager.addContentRestorer(this.contentRestorer);
+
         return glTFResource;
       });
     });
+
+    this._dispatchProgressEvent(undefined, promise);
+    return promise;
+  }
+
+  /**
+   * @internal
+   */
+  _dispatchProgressEvent(progress?: IProgress, taskPromise?: Promise<any>): void {
```

Comment:
This function should split two functions.

Reply:
done